src/components/common/ModalComponent.jsx

```diff
@@ -20,14 +20,14 @@ export default class ArtModal extends Component {
       <div className={this.props.className}>
         <Modal
           trigger={
-            (
-              <span tabIndex="-1" role="button" onClick={this.toggleModal} onKeyUp={(() => {})}>
-                { this.props.trigger }
+            this.props.trigger ? (
+              <span tabIndex="-1" role="button" onClick={this.toggleModal} onKeyUp={(() => { })}>
```

Comment:
OnKeyUp value should be `onKeyUp={() => {}}`. Note that there is no empty space between the return object of `() => {}`

Reply:
Done